src/execCmd.ts

```diff
@@ -303,3 +303,125 @@ export function execCmd<T = Collection>(
     }
   }
 }
+
+function toString(arrOrString: string | string[]): string {
```

Comment:
``` suggestion
function toString(arrOrString: Many<string>): string {
```

Reply:
ooh, `Many` is an unfortunate name for that